spec/values/numbers/modulo/floats.hrx

```diff
@@ -0,0 +1,45 @@
+<===> positive_positive/input.scss
+a {
+  b: +1.2 % +4.7;
```

Comment:
```suggestion
  b: 1.2 % 4.7;
```

I think including the explicit `+` here is a bit confusing—the numbers are still unambiguously positive without it.

Reply:
I agree about that. The explicit `+` was already there in the old spec and I copied it verbatim. I will update that